src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -442,8 +442,8 @@ public void addOnRankedMatchNotificationListener(Consumer<MatchmakerMessage> lis
             .thenRun(() -> {
               // TODO this should be sent by the server!
```

Comment:
we should remove the todo

Reply:
No. We still lack the team number and whatever the players argument does.

Reply:
Team number and players are being sent now. The players argument is just a number that tells the lobby it should wait until that many players have joined before launching the game.

Reply:
So u are argueing for removing it?